public/js/main.js

```diff
@@ -15,6 +15,12 @@ var acelaya = {
 
         $form.submit(function (e) {
             e.preventDefault();
+
+            // remove .alert block when it exists
+            if ($(".container .alert").length) {
+                $(".container .alert").remove();
+            }
```

Comment:
yeah, but the ".alert" element didn't created yet before form submitted.

Warm regards,

Abdul Malik Ikhsan

Pada 24 Jan 2015, pukul 18.35, Alejandro Celaya <notifications@github.com> menulis:

> In public/js/main.js:
> 
> > @@ -15,6 +15,12 @@ var acelaya = {
> >  
> >          $form.submit(function (e) {
> >              e.preventDefault();
> > +
> > +            // remove .alert block when it exists
> > +            if ($(".container .alert").length) {
> > +                $(".container .alert").remove();
> > +            }
> Wouldn't it be the same to just do this?
> 
> $(".container .alert").remove();
> —
> Reply to this email directly or view it on GitHub.
>